src/index.ts

```diff
@@ -20,8 +20,10 @@ function doCallback() {
   });
 }
 
+export type Subscribeable = Pick<Replicache, 'subscribe'>;
+
 export function useSubscribe<R extends ReadonlyJSONValue>(
-  rep: Replicache | null | undefined,
+  rep: Replicache | Subscribeable | null | undefined,
```

Comment:
Why do we need both Replicache and Subscribeable?

Reply:
Was debating this.  I think it makes it more obvious to customers that they can pass Replicache here... it still not obvious that they can pass ReflectClient.   

I think something like:
`subscribable: Replicache | ReflectClient | null | undefined `
might me more obvious.